osu.Game/Screens/Play/FailAnimation.cs

```diff
@@ -28,7 +28,7 @@ public class FailAnimation : Component
 
         private readonly DrawableRuleset drawableRuleset;
 
-        private readonly BindableDouble trackFreq = new BindableDouble(1);
```

Comment:
This should not be changed (avoid null values wherever possible). Instead just don't apply the transform.